content/docs/setup/kubernetes/spec-requirements/index.md

```diff
@@ -38,3 +38,67 @@ cluster must satisfy the following requirements:
 
 * _**`NET_ADMIN` capability**:_ If pod security policies are enforced in your cluster and unless you use the [Istio CNI Plugin](/docs/setup/kubernetes/istio-cni-install/), your pods must have the `NET_ADMIN` capability allowed.
 See [Required Pod Capabilities](/help/ops/setup/required-pod-capabilities/).
+
+* _**Check required ports**_: The following table lists the default ports used by Istio components. Make sure the ports are available before installing Istio components.
+
+    | Component | Port | NodePort | Description |
+    |----|----|----|----|
+    | Citadel | 8060 |   | The port number for Citadel GRPC server |
+    | | | | |
+    | Core DNS | 53 |   | DNS - TCP/UDP Protocols |
+    | Core DNS | 8053 |   | DNS plugin |
+    | Core DNS | 9153 |   | DNS |
+    | | | | |
+    | Egress gateway | 80 |   |   |
+    | Egress gateway | 443 |   |   |
+    | Egress gateway | 15090 |   | Proxy |
+    | Egress gateway | 15443 |   | This is the port where SNI routing happens |
+    | | | | |
+    | Galley | 443 |   | Validation port |
+    | Galley | 9091 |   | Mesh control Plane - Running mixer |
+    | Galley | 9094 |   | Port to use for exposing profiling (no longer used?) |
+    | Galley | 15014 |   | Monitoring |
+    | | | | |
+    | Grafana | 3000 |   | Grafana |
+    | Grafana | 15031 |   | Grafana |
+    | | | | |
+    | Envoy | 15000 |   | Envoy admin port (commands/diagnostics) |
+    | | | | |
+    | Ingress gateway | 80 | 31380 |   |
+    | Ingress gateway | 443 | 31390 |   |
+    | Ingress gateway | 15029 |   | Kiali |
+    | Ingress gateway | 15030 |   | Prometheus |
+    | Ingress gateway | 15031 |   | Grafana |
+    | Ingress gateway | 15032 |   | Tracing |
+    | Ingress gateway | 15090 |   | Proxy |
+    | Ingress gateway | 15443 |   | This is the port where SNI routing happens |
+    | Ingress gateway | 31400 | 31400 | Sample port|
+    | | | | |
+    | Mixer | 9091 |   | Policy/Telemetry |
+    | Mixer | 15004 |   | Policy/Telemetry - `mTLS`|
+    | Mixer | 15014 |   | Policy/Telemetry monitoring port |
+    | Mixer | 15090 |   | Proxy |
+    | Mixer | 42422 |   | Telemetry - Prometheus |
+    | | | | |
```

Comment:
could we separate default on component from default off component?

Could we use policy and telemetry as that is what user sees in k get services or deploy?

Reply:
I used Mixer since it's what the chart is named (that column is by chart name). I could change _Mixer_ to _Policy, Telemetry_ in to better coincide with the pod names if others feel that makes more sense.